lib/kino/smart_cell/chart_builder.ex

```diff
@@ -0,0 +1,180 @@
+defmodule Kino.SmartCell.ChartBuilder do
+  use Kino.JS, assets_path: "lib/assets/chart_builder"
+  use Kino.JS.Live
+  use Kino.SmartCell, name: "Chart builder"
+
+  @as_int ["width", "height"]
+  @as_atom ["data", "chart", "x_axis_type", "y_axis_type", "color_type"]
```

Comment:
Nit: it's better to keep those as strings and convert to atoms only when generating the code. This way we keep the same type as serialized/deserialized in `attrs` and there's gonna be less surprises :)